iot_config/src/org.rs

```diff
@@ -148,6 +333,7 @@ async fn insert_constraints(
         r#"
         insert into organization_devaddr_constraints (oui, net_id, start_addr, end_addr)
         values ($1, $2, $3, $4)
+        on conflict net_id do nothing
```

Comment:
I don't see any constraints on this table that could cause a conflict.

Reply:
dang; good catch